### PR TITLE
Inject informations into about dialog

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Dependent of:
* gridsuite/deployment#353
* gridsuite/study-server#484
* gridsuite/commons-ui#321

### CODE REVIEW

File: src/components/app-top-bar.js [hunk 1]
```diff
@@ -41,14 +47,6 @@ const AppTopBar = ({ user, userManager }) => {
...
     }, [user, dispatch]);
 
-    function hideParameters() {
-        setShowParameters(false);
-    }
-
-    function onLogoClicked() {
-        navigate('/', { replace: true });
-    }
-
```
Comment (EstherDarkish) [inline src/components/app-top-bar.js:51]: These changes are not required in this PR

**Review by EstherDarkish — APPROVED** [review]

File: decs.d.ts [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
         RadioGroupProps,
         TextFieldProps,
     } from '@mui/material';
+    import {TopBar as RawTopBar} from '@gridsuite/commons-ui';
+
+    export const DARK_THEME: String, LIGHT_THEME: String;
+
+    export const TopBar: FunctionComponent<InferProps<typeof RawTopBar.propTypes>>;
+
+    function logout(dispatch: unknown, userManagerInstance: unknown): Promise<unknown | undefined>;
```
Comment (flomillot) [inline decs.d.ts]: We already discussed about it and said we only use `any` and not `unknown`, no ?

**Review by EstherDarkish — COMMENTED** [review]
Error in the console : missing translation

`Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "close" for locale "en", using id as fallback.`

File: src/components/app-top-bar.js (hunk 1 above)
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:51]: I did like in gridstudy-app PR

File: decs.d.ts (hunk 2 above)
Comment (Tristan-WorkGH) [reply decs.d.ts]: :ok:

**Review by flomillot — APPROVED** [review]

**Pull request merged**